addon/components/o-s-s/button-dropdown.stories.js

```diff
@@ -49,20 +59,22 @@ export default {
 const defaultArgs = {
   icon: 'far fa-user',
   label: 'Actions',
+  square: false,
```

Comment:
Is it necessary to set "square" as false here since it's an optional argument?

Reply:
necessary, no, but it's fine as is.